Documentation/intro-to-corert.md

```diff
@@ -1,19 +1,20 @@
 Intro to .NET Native and CoreRT
 ===============================
 
-Native compilation is a great scenario addition to .NET Core apps on Windows, OS X and Linux. We've seen significant startup and throughput benefits of native compilation for Windows UWP apps, using .NET Native. Today, many native apps and tools benefit from being compiled by a C++ compiler, and not as much by being written in C++. .NET Native brings much of the performance and all of the deployment benefits of native compilation, while retaining your ability to write in your favorite .NET programming language.
+Native (AOT) compilation is a great scenario addition to .NET Core apps on Windows, OS X and Linux. We've seen significant startup and throughput benefits of native compilation for Windows UWP apps, using .NET Native. Today, many native apps and tools benefit from being compiled by a C++ compiler, and not as much by being written in C++. CoreRT brings much of the performance and all of the deployment benefits of native compilation, while retaining your ability to write in your favorite .NET programming language.
 
 Architecture
 ============
 
-[.NET Native](https://msdn.microsoft.com/library/dn584397.aspx) is a native  toolchain that compiles [CIL byte code](https://en.wikipedia.org/wiki/Common_Intermediate_Language) to machine code (e.g. X64 instructions). By default, .NET Native (for .NET Core, as opposed to UWP) uses RyuJIT as an ahead-of-time (AOT) compiler, the same one that CoreCLR uses as a just-in-time (JIT) compiler. It can also be used with other compilers, such as [LLILC](https://github.com/dotnet/llilc), UTC for UWP apps and [IL to CPP](https://github.com/dotnet/corert/tree/master/src/ILCompiler.CppCodeGen/src/CppCodeGen) (an IL to textual C++ compiler we have built as a reference prototype).
+[CoreRT](https://github.com/dotnet/corert) (for .Net Core) and [.NET Native](https://msdn.microsoft.com/library/dn584397.aspx) (for UWP) are native toolchains that compile [CIL byte code](https://en.wikipedia.org/wiki/Common_Intermediate_Language) to machine code (e.g. X64 instructions). By default, CoreRT uses RyuJIT as an ahead-of-time (AOT) compiler, the same one that CoreCLR uses as a just-in-time (JIT) compiler. CoreRT can also be used with other compilers, such as [LLILC](https://github.com/dotnet/llilc), and [IL to CPP](https://github.com/dotnet/corert/tree/master/src/ILCompiler.CppCodeGen/src/CppCodeGen) (an IL to textual C++ compiler we have built as a reference prototype). .NET Native uses the UTC compiler.
```

Comment:
This makes is sound like .NET Native for UWP is completely different different from CoreRT. That's not accurate. .NET Native for UWP is using CoreRT as one of its parts, i.e.:

.NET Native = CoreRT + UTC + other UWP-specific parts.

I would just say: [CoreRT](https://github.com/dotnet/corert) is a toolchain that compiles ... . And keep the mention of .NET Native (for UWP) and UTC at the end of the paragraph.